tests/unit/acl.tcl

```diff
@@ -465,6 +465,33 @@ start_server {tags {"acl external:skip"}} {
         }
     }
 
+    test "ACL CAT with illegal arguments" {
+        assert_error {*Unknown category 'NON_EXISTS'} {r ACL CAT NON_EXISTS}
+        assert_error {*Unknown subcommand or wrong number of arguments for 'CAT'*} {r ACL CAT NON_EXISTS NON_EXISTS2}
+    }
+
+    test "ACL CAT without category - list all categories" {
+        set reply [r acl cat]
+        assert_match "*keyspace*" $reply
+        assert_match "*connection*" $reply
```

Comment:
rather than string matching, i rather convert it to a list, and look up an element.

Reply:
using lsearch